judge/views/organization.py

```diff
@@ -312,3 +312,8 @@ def post(self, request, *args, **kwargs):
 
         organization.members.remove(user)
         return HttpResponseRedirect(organization.get_users_url())
+
+
```

Comment:
I think this deserves some explanation as to why we have this, before we forget about this migration in years to come.